cmd/coordinator/cmd/utils.go

```diff
@@ -170,6 +170,21 @@ func (c *coordinator) setupVeth(logger *zap.Logger, containerID string) error {
 		if err := netlink.LinkSetUp(link); err != nil {
 			return fmt.Errorf("failed to set %q UP: %v", containerInterface.Name, err)
 		}
+
+		if c.ipFamily == netlink.FAMILY_V6 {
+			// set an address to veth to fix work with istio
+			// set only when not ipv6 only
+			return nil
+		}
+
+		if err = netlink.AddrAdd(link, &netlink.Addr{
```

Comment:
I suggest add some comments here to explain and attach the issue link

Reply:
done